composer.json

```diff
@@ -36,6 +36,7 @@
     },
     "require-dev": {
         "ext-curl": "*",
+        "ext-openssl": "*",
```

Comment:
Library itself does not need this extension, only tests.

Reply:
Hence require-dev.